msg/RadarTrack.msg

```diff
@@ -0,0 +1,12 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+uint16 track_id                             # The track ID of the object generated by the radar.
+
+                                            # NB: The z component of these fields is ignored for 2D tracking.
+geometry_msgs/Point position                # x, y, z coordinates of the centroid of the object being tracked.
+geometry_msgs/Vector3 velocity              # The velocity of the object in each spacial dimension.
+geometry_msgs/Vector3 acceleration          # The acceleration of the object in each spacial dimension.
+geometry_msgs/Vector3 size                  # The average deviation (m) in the x, y, z, dimensions
+                                            #   between the center of the object and the data points which it is derived from.
+string classification                       # An optional classification of the object (eg. Person, Car).
```

Comment:
I second this. In my limited radar experience, I've only ever seen things classified by "movement". Meaning an object is classified as either "stationary/static" or "moving/dynamic".

Reply:
It is possible to make some determination about what type of object is being detected based on the signature of the data and some drivers could/do implement this. The AUTOLIV 77GHz FLR RADAR driver has 'object_type' for example. I think it would be good to allow for this valuable information to be populated if it is available.

In regards to an ENUM. As far as I know there is no such list we could draw from. I very much expect the classification will be manufacturer and application specific. Using string allows the field to be used in an application specific way, while still maintaining the common message format. I think this is a good compromise.

Reply:
OK that sounds like a reasonable compromise.